app/kuma-dp/pkg/dataplane/envoy/remote_bootstrap_test.go

```diff
@@ -81,81 +62,59 @@ var _ = Describe("Remote Bootstrap", func() {
 			_, err = writer.Write(responseBytes)
 			Expect(err).ToNot(HaveOccurred())
 		})
-		port, err := strconv.Atoi(strings.Split(server.Listener.Addr().String(), ":")[1])
-		Expect(err).ToNot(HaveOccurred())
 
 		// and
-		generator := NewRemoteBootstrapGenerator("linux")
+		bootstrapClient := NewRemoteBootstrapClient("linux")
 
 		// when
-		bootstrap, kumaSidecar, err := generator(context.Background(), fmt.Sprintf("http://localhost:%d", port), given.config, given.params)
+		bootstrap, kumaSidecar, err := bootstrapClient.Fetch(
+			context.Background(),
+			given.optsBuilder.cpURL(server.URL).build(),
+			given.metadata,
+		)
 
 		// then
 		Expect(err).ToNot(HaveOccurred())
 		Expect(bootstrap).ToNot(BeNil())
 		Expect(kumaSidecar).ToNot(BeNil())
 	},
-		Entry("should support port range with exactly 1 port",
```

Comment:
are these no longer needed?

Reply:
These didn’t make sense, there’s no port range involved. I updated the descriptions, removed the duplicate, and improved the golden file names.